Unix/installbuilder/service_scripts/service_control.unix

```diff
@@ -143,6 +155,7 @@ restart_omi()
 
     if [ "$TYPE" = "A" -o "$TYPE" = "H" -o "$TYPE" = "S" ]; then
 	stop_omi
+	wait_service_stop
```

Comment:
Since this is only called from one place, you didn't need to make it a subroutine. But this is a style choice, I have no problem with the change. Signing off.